openmc/data/decay.py

```diff
@@ -570,3 +574,48 @@ def sources(self):
 
         self._sources = merged_sources
         return self._sources
+
+
+_DECAY_PHOTON_ENERGY = {}
+
+
+def decay_photon_energy(nuclide: str) -> Univariate:
+    """Get photon energy distribution resulting from the decay of a nuclide
+
+    This function relies on data stored in a depletion chain. Before calling it
+    for the first time, you need to ensure that a depletion chain has been
+    specified in openmc.config['chain_file'].
+
+    .. versionadded:: 0.14.0
+
+    Parameters
+    ----------
+    nuclide : str
+        Name of nuclide, e.g., 'Co58'
+
+    Returns
+    -------
+    openmc.stats.Univariate
+        Distribution of energies in [eV] of photons emitted from decay. Note
+        that the probabilities represent intensities, given as [decay/sec].
+    """
+    if not _DECAY_PHOTON_ENERGY:
+        chain_file = openmc.config.get('chain_file')
+        if chain_file is None:
+            raise DataError(
+                "A depletion chain file must be specified with "
+                "openmc.config['chain_file'] in order to load decay data."
+            )
```

Comment:
Thought this was easier to parse:

```suggestion
    if not _DECAY_PHOTON_ENERGY and openmc.config.get('chain_file') is None:
        raise DataError(
            "A depletion chain file must be specified with "
            "openmc.config['chain_file'] in order to load decay data."
        )
```

Reply:
Oh whoops, yeah that's in the same block. My bad!